app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,31 @@
+<div>
```

Comment:
- [ ] Whenever a user writes a wrong email or password, you should flash a message indicating that
![image](https://user-images.githubusercontent.com/62705947/190195334-25874b83-bad8-4a09-bf9b-7254c7a8a8a0.png)

Reply:
can you help me in this?

Reply:
Hi @Fatima-hub333,
To do this requirement please check error handling partials rendered in this line 
https://github.com/Fatima-hub333/Capstone-aka-Budget-App/blob/2d0cec7bafbb60d428a58008e92dae6d4a98ef1e/app/views/devise/registrations/new.html.erb#L6
and this file too [app](https://github.com/Fatima-hub333/Capstone-aka-Budget-App/tree/featured/app)/[views](https://github.com/Fatima-hub333/Capstone-aka-Budget-App/tree/featured/app/views)/[layouts](https://github.com/Fatima-hub333/Capstone-aka-Budget-App/tree/featured/app/views/layouts)/application.html.erb and you can see this [video ](https://www.youtube.com/watch?v=RQQ98mbh2Ic)too